libs/wire-api/src/Wire/API/Event/FeatureConfig.hs

```diff
@@ -47,20 +47,20 @@ instance Arbitrary Event where
     do
       let arbConfig =
             oneof
-              [ arbitrary @(WithStatus SSOConfig) <&> toJSON,
-                arbitrary @(WithStatus SearchVisibilityAvailableConfig) <&> toJSON,
-                arbitrary @(WithStatus ValidateSAMLEmailsConfig) <&> toJSON,
-                arbitrary @(WithStatus DigitalSignaturesConfig) <&> toJSON,
-                arbitrary @(WithStatus AppLockConfig) <&> toJSON,
-                arbitrary @(WithStatus FileSharingConfig) <&> toJSON,
-                arbitrary @(WithStatus ClassifiedDomainsConfig) <&> toJSON,
-                arbitrary @(WithStatus ConferenceCallingConfig) <&> toJSON,
-                arbitrary @(WithStatus SelfDeletingMessagesConfig) <&> toJSON,
-                arbitrary @(WithStatus GuestLinksConfig) <&> toJSON,
-                arbitrary @(WithStatus SndFactorPasswordChallengeConfig) <&> toJSON,
-                arbitrary @(WithStatus SearchVisibilityInboundConfig) <&> toJSON,
-                arbitrary @(WithStatus MLSConfig) <&> toJSON,
-                arbitrary @(WithStatus ExposeInvitationURLsToTeamAdminConfig) <&> toJSON
+              [ arbitrary @(LockableFeature SSOConfig) <&> toJSON,
+                arbitrary @(LockableFeature SearchVisibilityAvailableConfig) <&> toJSON,
+                arbitrary @(LockableFeature ValidateSAMLEmailsConfig) <&> toJSON,
+                arbitrary @(LockableFeature DigitalSignaturesConfig) <&> toJSON,
+                arbitrary @(LockableFeature AppLockConfig) <&> toJSON,
+                arbitrary @(LockableFeature FileSharingConfig) <&> toJSON,
+                arbitrary @(LockableFeature ClassifiedDomainsConfig) <&> toJSON,
+                arbitrary @(LockableFeature ConferenceCallingConfig) <&> toJSON,
+                arbitrary @(LockableFeature SelfDeletingMessagesConfig) <&> toJSON,
+                arbitrary @(LockableFeature GuestLinksConfig) <&> toJSON,
+                arbitrary @(LockableFeature SndFactorPasswordChallengeConfig) <&> toJSON,
+                arbitrary @(LockableFeature SearchVisibilityInboundConfig) <&> toJSON,
+                arbitrary @(LockableFeature MLSConfig) <&> toJSON,
+                arbitrary @(LockableFeature ExposeInvitationURLsToTeamAdminConfig) <&> toJSON
```

Comment:
I don't think `hcFoldMap` can be used here, since we don't have any NP or SOP to fold over. I changed this so that it uses the `Features` list by explicit recursion.